dts/arm/stm32l476g_disco.dts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2017 Arthur Sfez
+ *
+ * Based on stm32l496g_disco:
+ *
+ * Copyright (c) 2017 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <st/stm32l476.dtsi>
+
+/ {
+	model = "STMicroelectronics STM32L476G-DISCO board";
+	compatible = "st,stm32l476g-disco", "st,stm32l476";
+
+	chosen {
+		zephyr,console = &usart2;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+	};
+};
+
+&usart1 {
```

Comment:
Looks like this is missing from the pinmux. Am I missing something?

Reply:
All the other pins USART's pins are used by something else in the discovery board, so I removed the usart1 entry from the DTS file. Is it the appropriate way to handle this?

Reply:
That seems reasonable to me.

Reply:
Alright. If I wanted to improve the board support, would it be interesting to add in the dts/pinmux every port/pin definition (i2c, spi, etc.) that are used by the on-board peripherals (joystick, flash memory, etc) to make things easier for an user that would want to build an application for this board? Or is that overkill and the console/led is enough here?

Reply:
AFAIK adding peripherals that are used, especially by on-board peripherals, is always a good thing. But they need to be added to the pinmux too or it might be confusing why they don't work, IMO.